test/integration/maker/UtilityMaker.t.sol

```diff
@@ -61,7 +61,7 @@ contract MakerUtilityTest is Test, MakerCommonUtils, ERC20Permit2Utils {
         IMakerVat vat = IMakerVat(VAT);
         bytes32 ilkETH = bytes32(bytes(ETH_JOIN_NAME));
         (, uint256 rate, uint256 spot, , uint256 dust) = vat.ilks(ilkETH);
-        (uint256 daiDrawMin, uint256 minCollateral) = _getDAIDrawMinAndMinCollateral(spot, dust);
+        (uint256 daiDrawMin, uint256 minCollateral) = _getDAIDrawMinAndMinCollateral(spot, dust, 18);
```

Comment:
Since the last parameter is the decimal of collateral, is it the same as the GEM_DECIMAL?

Reply:
It's collateral decimal, and in here is the LockETH case so is 18 (GEM_DECIMAL  for LockGem)

Reply:
I see. In that case I think defining the constant just like GEM_DECIMAL will be better for understanding.

Reply:
like ETH_DECIMAL for LockETH case~?

Reply:
Updated!